src/peft/utils/other.py

```diff
@@ -583,8 +582,12 @@ def __init__(
         module_to_save: torch.nn.Module,
         adapter_name: str,
         token_indices: list[int],
+        tied_adapter=None,
     ) -> None:
-        super().__init__(module_to_save, adapter_name, token_indices=token_indices)
+        """Supports weight-tying to another adapter when passed a `tied_adapter` which is expected to be a
+        `TrainableTokensLayer`.
+        """
```

Comment:
Should be a comment?

Reply:
I moved it to the class docstring but I disasgree that it should be a comment.